URBNSwiftAlert/Classes/AlertConfiguration.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AlertConfiguration.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 5/22/17.
+//
+//
+import Foundation
+
+public struct AlertConfiguration {
+    /**
+     *  Title text for the alert
+     */
+    public var title: String?
+    
+    /**
+     *  Message text for the alert
+     */
+    public var message: String?
+    
+    /**
+     *  The view to present from when using showInView:
+     */
+    public var presentationView: UIView?
+    
+    /**
+     *  Flag if the alert is active. False = a passive alert
+     */
+    public var isActiveAlert = false
+    
+    /**
+     *  Duration of a passive alert (no buttons added)
+     */
+    public var duration: CGFloat?
```

Comment:
we could remove `isActiveAlert` _(or make it a read-only computed property)_ and define `isActiveAlert` when this is `nil`. That way we don’t have 2 variables essentially controlling the state of the alert

Reply:
isActiveAlert now computed